src/sorting.cpp

```diff
@@ -133,3 +133,45 @@ std::vector<int> counting_sort(std::vector<int>& vec, int k) {
 
   return result;
 }
+
```

Comment:
I would like to see a litle explenation of the function befor the implementation as the other sorting-funtions have. With just maby som information about the sort, (as this repo is for learning, but it is not nesserary)